lang/nhc98/Portfile

```diff
@@ -1,32 +1,31 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
-PortSystem  1.0
+PortSystem      1.0
 
-name        nhc98
-version     1.22
-revision    0
-categories  lang
-maintainers nomaintainer
+name            nhc98
+version         1.22
+revision        1
+categories      lang
+maintainers     Sergey Fedorov @barracuda156
```

Comment:
this is an incorrect `maintainers` line; it should provide an (obfuscated) e-mail address and a GitHub handle, and ideally also `openmaintainer`

Reply:
@reneeotten Thank you! How to write e-mail (gmail) and Github handle correctly?
And you are right, `openmaintainer` should be added, I just forgot about it.